10 - Loops/the_next_keyword.rb

```diff
@@ -0,0 +1,45 @@
+# This function finds all occurrences of the '$' sign in the given sentence using a simple while loop.
+# It prints the index of each '$' found in the string.
+
+def find_dollars_while(money_sentence)
+  current_index = 0 
+  final_index = money_sentence.length - 1 
+  
+  while current_index <= final_index 
+    if money_sentence[current_index] == "$" 
+      puts "Found $ at index #{current_index}" 
+    end
+    current_index += 1 
+  end
+
+  # The loop will terminate once all characters have been checked.
+end
+
+
+money_sentence = "I love $ in the morning, $ in the afternoon, and $ at night"
+
+puts "Find Dollar method result using while loop"
```

Comment:
Task is to find dollar sign index using while loop so make statement accordingly
`puts "Dollar sign positions (using while loop):"`